subsystems/shootersubsystem.py

```diff
@@ -75,7 +72,19 @@ def periodic(self) -> None:
         self.initializationMaximum = max(
             self.initializationMaximum, self.turretMotor.getPosition()
         )
-        if not SmartDashboard.getBoolean(constants.kShootingManualModeKey, False):
+        SmartDashboard.putNumber(
```

Comment:
Do we really need this?

Reply:
This is used during turret range calibration. If we truly don't need it can remove when we change SmartDashboard to NT property